beets/dbcore/query.py

```diff
@@ -152,16 +152,17 @@ class NoneQuery(FieldQuery):
 
     def __init__(self, field, fast=True):
         super(NoneQuery, self).__init__(field, None, fast)
+        self.pattern = None
 
     def col_clause(self):
         return self.field + " IS NULL", ()
 
-    @classmethod
-    def match(cls, item):
-        try:
-            return item[cls.field] is None
-        except KeyError:
-            return True
+    def match(self, item):
+        return self.value_match(self.pattern,
+                                item.get(self.field, default=None))
+
+    def value_match(self, pattern, value):
+        return value is pattern
```

Comment:
I think this is ok. Also becuause the call from the match method also uses the `self.patern` initiated in the __init__.
Again, I am to new to Python to tell. Something like this in Php, not implementing an abstract class would fail at compile time... but then again this is not php. So your call: should I remove it?